app/pages/_document.tsx

```diff
@@ -4,10 +4,7 @@ import { GA_TRACKING_ID } from "../domain/env";
 class MyDocument extends Document {
   render() {
     return (
-      <Html
-        data-app-version={`${process.env.NEXT_PUBLIC_VERSION}`}
-        lang={this.props.locale}
-      >
+      <Html data-app-version={`${process.env.NEXT_PUBLIC_VERSION}`}>
```

Comment:
Since in later commits the setting of the lang is removed, why don't we keep it here ?

Reply:
Understood, this is done automatically by next.js now, settings it directly on document is deprecated.